.github/workflows/colp_build.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       run: ./bash/build_env_setup.sh
 
     - name: Dataloading ...
-      run: python -m dcpy.builds.load
+      run: python -m dcpy.builds.load recipe
```

Comment:
We need to add these because of a fun `typer` behavior where if there's only one command defined, it just ignores the app command, in this case the `load` in `@app.command("load")`.

Reply:
I've come across this a few times and I HATE IT SO MUCH